micro-infra-spring-config/src/main/java/com/ofg/infrastructure/property/FileSystemLocator.java

```diff
@@ -77,7 +78,12 @@ public ConfigLocations getConfigLocations() {
     private Map<String, Object> decrypt(Map<String, Object> sourceMap) {
         final Map<String, Object> result = new HashMap<String, Object>();
         for (Map.Entry<String, Object> entry : sourceMap.entrySet()) {
-            result.put(entry.getKey(), decryptIfEncrypted(entry.getValue()));
+            try {
+                result.put(entry.getKey(), decryptIfEncrypted(entry.getValue()));
+            } catch (Exception e) {
+                log.error(String.format("Exception occurred while trying to decrypt key [%s]", entry.getKey()), e);
```

Comment:
yes we wanted to do it with `{}` but once you want to have both the `Throwable` and the object then it's not really possible if I'm not mistaken.